srml/council/src/seats.rs

```diff
@@ -680,15 +680,15 @@ impl<T: Trait> Module<T> {
 	///
 	/// The voter index must be provided as explained in [`voter_at`] function.
 	fn do_set_approvals(who: T::AccountId, votes: Vec<bool>, index: VoteIndex, hint: SetIndex) -> Result {
-		let candidates = Self::candidates();
+		let candidates_len = <Self as Store>::Candidates::len().expect("vec always has a len; qed");
```

Comment:
As with the other decoding errors, we should probably hide it behind the scenes and just panic there.

Reply:
decoding should not panic in the runtime; it should return `None` behind the scenes, which gets coerced into `Default::default()` for items that are not `Option`s.